srcpkgs/typstyle/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'typstyle'
+pkgname=typstyle
+version=0.11.27
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="libgit2-devel"
+depends="git"
```

Comment:
This is only used in the build script to obtain git info that isn't there anyway in the distfile. We should patch out all usage of git from the build deps, scripts and source, or even better, convince upstream to feature-gate it or sth like that.

Reply:
Thank you for upstreaming the changes!